src/rocm_memory.c

```diff
@@ -44,9 +45,13 @@ static int init_rocm(int device_id) {
 
 	hipDeviceProp_t prop = {0};
 	ROCM_CHECK(hipGetDeviceProperties(&prop, device_id));
+#if HIP_VERSION >= 60000000
+	printf("Using ROCm Device with ID: %d, Name: %s, PCI Bus ID: 0x%x, GCN Arch: %s\n",
+	       device_id, prop.name, prop.pciBusID, prop.gcnArchName);
```

Comment:
nit: IMHO it would be cleaner to use some local variable for gcnArch / gcnArchName and keep a single printf.

Reply:
the problem is that one is an integer(gcnArch), the other a string (gcnArchName). If you want I can print the integer into a string to unify the printf statement, would you prefer that solution?

Reply:
You can also split the print. I'm ok with either way.